dev/travis/js/config.php

```diff
@@ -0,0 +1,360 @@
+<?php
```

Comment:
The documentation in question is about generating static on a non-production machine, so this is a slightly different case. Providing a hard-coded configuration file is not the best practice and needs to be maintained all the time.

I would still suggest installing application for now, at least for consistency, since internally this is what we do to run Jasmine tests.